katas/content/nonlocal_games/index.md

```diff
@@ -53,9 +51,147 @@ To start with, let's take a look at how you would play the classical variant of
     "path": "./chsh_classical_strategy/"
 })
 
+@[exercise]({
+    "id": "nonlocal_games__chsh_quantum_alice_strategy",
+    "title": "Alice's Quantum Strategy",
+    "path": "./chsh_quantum_alice_strategy/"
+})
+
+@[exercise]({
+    "id": "nonlocal_games__chsh_quantum_bob_strategy",
+    "title": "Bob's Quantum Strategy",
+    "path": "./chsh_quantum_bob_strategy/"
+})
+
 @[section]({
-    "id": "nonlocal_games__conclusion", 
-    "title": "Conclusion" 
+    "id": "nonlocal_games__discussion",
+    "title": "Discussion: Probability of Victory for Quantum Strategy"
+})
+
+The above quantum strategy adopted by Alice and Bob offers a win rate of $\frac{2 + \sqrt{2}}{4}$, or about 85.36%. Let's see why this is the case.
+
+First, consider the outcome if Alice and Bob simply measure their qubits in the Z basis without manipulating them at all. Because of the entanglement inherent to the Bell state they hold, their measurements will always agree (i.e., both true or both false).
+This will suffice for victory in the three scenarios (0,0), (0,1) and (1,0) and fail for (1,1), so their win probability is 75%, the same as that for the straightforward classical strategies of invariably returning both true or both false.
+
+Now let's analyze the optimal quantum strategy.
+
+> As a reminder, probability "wavefunction" for a two-qubit state is given by the following length-4 vector of amplitudes:
+> $$\begin{bmatrix}\psi_{00}\\\psi_{01}\\\psi_{10}\\\psi_{11}\end{bmatrix}$$
+> $|\psi_{ij}|^2$ gives the probability of observing the corresponding basis state $|ij\rangle$ upon measuring the qubit pair.
+
+The initial state $|00\rangle$ has $\psi_{00} = 1$ and $\psi_{01} = \psi_{10} = \psi_{11} = 0$.
+The Bell state we prepare as the first step of the game has an amplitude vector as follows (we'll use decimal approximations for matrix elements):
+
+$$\begin{bmatrix}1/\sqrt{2}\\0\\0\\1/\sqrt{2}\end{bmatrix} = \begin{bmatrix}0.7071\\0\\0\\0.7071\end{bmatrix}$$
+
+Let's analyze the probabilities of outcomes in case of different bits received by players.
+
+## Case 1: Alice holds bit 0
+
+In this case Alice simply measures in the Z basis as above.
+
+- When Bob's bit is 0, he rotates his qubit clockwise by $\pi/8$, which corresponds to the operator
+
+$$\begin{bmatrix}
+    0.9239 & 0.3827 & 0 & 0\\
+    -0.3827 & 0.9239 & 0 & 0\\
+    0 & 0 & 0.9239 & 0.3827\\
+    0 & 0 & -0.3827 & 0.9239
+\end{bmatrix}$$
+This performs the $R_y$ rotation by $\pi/8$ radians clockwise on Bob's qubit while leaving Alice's qubit unchanged.
```

Comment:
I think these two lines need to be indented so that they're treated as part of the first bullet point rather than separate paragraphs, and the matrix in the second bullet point as well

Reply:
Indentation is added.
Screen shot is attached.
![index_indentation](https://github.com/user-attachments/assets/839e2009-81a8-4804-b6eb-4eb5ab1d58fd)